lambdas/post_to_slack/post_to_slack.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+
+"""
+
+import json
+import pprint
+import os
+from botocore.vendored import requests
+
+
+def main(event, _):
+    print(f'Received event:\n{pprint.pformat(event)}')
+    message = event['Records'][0]['Sns']['Message']
+    message_data = json.loads(message)
+    alarm_name = message_data['AlarmName']
+    namespace = message_data['Trigger']['Namespace']
+    metric_name = message_data['Trigger']['MetricName']
+    dimensions = message_data['Trigger']['Dimensions']
+    state_reason = message_data['NewStateReason']
+
+    slack_data = {'username': 'cloudwatch-alert',
```

Comment:
I wasn't sure whether it would make sense to encapsulate this logic in a class, but this Lambda is so succinct as it is it's probably unnecessary.